src/routes/tekenmethodes/[slug]/stappenplan/+page.svelte

```diff
@@ -39,132 +13,90 @@
 
 <MethodsHeader {data} />
 
-
-<article class="grid">
-  <div class="flex-s">
-    <div class="tags-b">
-
-      <img class="icon" src="/icon1.svg" alt="icon" />
-      <div class="wrapper-methods">
-
-      <img class="icon" src="/images/category.svg" alt="icon" />
-
-      {#each data.methods as method}
-        {#each method.categories as category}
-          <h3 class="tag">
-            {@html category.title}
-          </h3>
+<section class="grid">
+  <div class="tags-container">
+   
+    <div class="tags-group">
+        <img class="icon" src="/images/category.svg" alt="icon" />
+        {#each data.methods as method}
+          {#each method.categories as category}
+            <p class="tag">{@html category.title}</p>
+          {/each}
         {/each}
-      {/each}
-    </div>
     </div>
 
-    <div class="tags-b">
+    <div class="tags-group">
       <img class="icon" src="/images/materials.svg" alt="icon" />
-      <h3 class="tag">
-        {#each data.methods as method}
-          {method.material}
-        {/each}
-      </h3>
+      {#each data.methods as method}
+        <p class="tag">{method.material}</p>
+      {/each}
     </div>
 
-    <div class="tags-b">
+    <div class="tags-group">
       <img class="icon" src="/images/duration.svg" alt="icon" />
-      <h3 class="tag">
-        {#each data.methods as method}
-          {method.duration}
-        {/each}
-      </h3>
+      {#each data.methods as method}
+        <p class="tag">{method.duration}</p>
+      {/each}
     </div>
 
-    <div class="tags-b">
+    <div class="tags-group">
       <img class="icon" src="/images/author.svg" alt="icon" />
-      <h3 class="tag">Charley Muhren</h3>
+      <p class="tag">Charley Muhren</p>
     </div>
   </div>
+  
 
-  <section class="steps">
+  <div class="steps">
     {#each data.methods as method}
       {#each method.steps as step}
         <details class="accordion-item">
           <summary class="step-title">{step.title}</summary>
           <div class="accordion-content">
-            <!-- {#each step.visual as visual} -->
-
             {#if step.visual[0]}
-              <div class="step-visual">
```

Comment:
Zou je hier ook een picture element kunnen gebruiken ipv een div?
Dan zijn er ook mogelijkheden om responsive images toe te passen

Reply:
Ik denk dat met mijn html structuur de div niet echt vervangbaar is met een picture!